letsencrypt/views.py

```diff
@@ -27,11 +24,5 @@ def detail(request, acme_data):
         AcmeChallenge,
         challenge=acme_data,
     )
-
-    return render(
-        request,
-        'letsencrypt/detail.html',
-        {
-            'response': acme_challenge.response
-        },
-    )
+    
+    return HttpResponse(acme_challenge.response, content_type="text/plain")
```

Comment:
Yeah I think I like this format a lot better, means a template should not be necessary at all!